### ISSUE TITLE
how to produce exact same byte count totals as du

### ISSUE THREAD

**Issue by omac777** [solidiquis/erdtree#144]
Firstly, I want to say I find your app very useful and cool.  Cheers to that.

I want to place confidence in erdtree for listing files/directories totals, but my issue with it is I cannot achieve the same totals as what I am used to seeing with du -cab which for me is a gold standard.   The byte count is different when I list erdtree itself du vs erdtree.  Why is that?

### erd

```
time ./target/release/erd --hidden --no-color ~/erdtree/
20463616 B erdtree
36 directories, 110 files
real	0m0.006s

```
### versus du

```
time du -cab ~/erdtree/
233522328	total
real	0m0.017s
```

How do I produce different **du** usages with **erdtree** with identical results(byte count totals)?  It would be useful to have such a table.

Thank you for reading this far.

**Comment by ilyapopov** [comment]
You need at least `-no-ignore` and `-follow`. This will get you close, but still not exact.

**Comment by ilyapopov** [comment]
Also, flag `-b` for `du` implies `--apparent-size`, which should be equivalent to `--disk-usage logical` for `erd`, but result is still not identical.

**Comment by Vollbrecht** [comment]
i have a large directory. Running with `erd -i.HL 1 -u=si -d=physical` i get the following report of the size of the dir.
```
 40.97 GB esp_rust

292 directories, 320 files
```
using `du -hls ./` in that same dir i got 
```
47G     ./
```
when i am using the -d=logical erdtree even reports less.
 Thanks for your great tool anyway!  Because of the tool i started more to think about how tools measure disk-sizes. Its similar in that regard to tools like top, htop etc and the question how much ram is in use currently systemwide. Every tool gives you a different answer, depending of an internal definition of what it is checking.

**Comment by ilyapopov** [comment]
`-l` flag for `du` makes it count hardlinked files multiple times, while `erd` [counts them only once](https://github.com/solidiquis/erdtree#hardlinks). I don't see a flag in `erd` to change this behavior.

**Comment by solidiquis** [comment]
@ilyapopov that seems to be an issue. I will address this.

**Comment by solidiquis** [comment]
Closed by https://github.com/solidiquis/erdtree/pull/172

This is going to be included in the v3.0 release tonight.

[Here's how to get the same output as du](https://github.com/solidiquis/erdtree#matching-du-output)
